qa-tests-backend/src/test/groovy/DefaultPoliciesTest.groovy

```diff
@@ -80,14 +87,14 @@ class DefaultPoliciesTest extends BaseSpecification {
 
     static final private Deployment STRUTS_DEPLOYMENT = new Deployment()
             .setName(STRUTS)
-            .setImage("quay.io/rhacs-eng/qa:struts-app")
+            .setImage(STRUTS_IMAGE)
             .addLabel("app", "test")
             .addPort(80)
 
     static final private List<Deployment> DEPLOYMENTS = [
         new Deployment()
             .setName (NGINX_LATEST)
-            .setImage ("quay.io/rhacs-eng/qa:latest") // this is docker.io/nginx:1.22-alpine but tagged as latest
+            .setImage ("quay.io/rhacs-eng/qa-multi-arch-nginx:latest")
```

Comment:
Can you include the comment that was removed? Is this still nginx:1.22-alpine?

Reply:
quay.io/rhacs-eng/qa-multi-arch-nginx:latest is version 1.23.3. Will readd the comment